docs/kubernetes/mailu/configmap.yaml

```diff
@@ -40,11 +40,20 @@
     TLS_FLAVOR: "cert"
 
     # Authentication rate limit (per source IP address)
-    AUTH_RATELIMIT: "10/minute;1000/hour"
+    AUTH_RATELIMIT: "100/minute;10000/hour"
```

Comment:
Why change this specifically for kubernetes?

Reply:
Long story, I created 2 e-mail addresses and connecting through IMAP on my laptop and mobile gave me login errors. It seemed that IMAP opens up multiple connections to the Dovecot server.
Although I have some strange behavior now: 
My laptop (apple mail, which is normally very stable) asks me for my password all the time (sometimes it works, sometimes it doesn't) and in the log files I see something like `[AUTHENTICATIONFAILED]`. This is rather bothering and I don't see a solution yet for this. I need some time to investigate more.

Reply:
BTW, any hints on this are always welcome, I really now the entered passwords are 100% correct